tests/spread/plugins/v2/catkin-ros1-catkin-packages-ignore/task.yaml

```diff
@@ -0,0 +1,39 @@
+summary: >-
+  Exercise the catkin-packages-ignore option of catkin plugin
+
+kill-timeout: 180m
+
+environment:
+  SNAP/catkin_catkin_packages_ignore: catkin-catkin-packages-ignore
+  SNAPCRAFT_ENABLE_EXPERIMENTAL_EXTENSIONS: "1"
+
+systems:
+  - ubuntu-20.04
+  - ubuntu-20.04-64
+  - ubuntu-20.04-amd64
+  - ubuntu-20.04-arm64
+
+prepare: |
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  set_base "../snaps/$SNAP/snap/snapcraft.yaml"
+
+restore: |
+  cd "../snaps/$SNAP"
+  snapcraft clean
+  rm -f ./*.snap
+
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  restore_yaml "snap/snapcraft.yaml"
+
+execute: |
+  cd "../snaps/$SNAP"
+
+  # Build the snap and verify that only package2 was included
+  snapcraft
+  [ -d parts/test-part/install/lib/package2 ]
+  [ ! -d parts/test-part/install/lib/package1 ]
+
+  snap install "${SNAP}"_1.0_*.snap --dangerous
+  [ "$($SNAP)" = "I am package2" ]
```

Comment:
Couple of these files have no trailing newline for whatever reason..

Reply:
Are you saying you want them?

Reply:
Doesn't matter, just a nit.